src/icon-buttons/icon-button.component.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default function IconButton(props) {
+  return (
+    <div
+      className={props.className}
```

Comment:
I think we could do better here - it should be a `<button>` instead of a `<div>` for accessibility purposes. And it should have a CSS class on it that we write that implements a normal and a hover state.